### ISSUE TITLE
Keyboard navigation in editor cannot reach top with toolbar_sticky

### ISSUE THREAD

**Issue by izmeez** [tinymce/tinymce#11246]
## 📝 Provide detailed reproduction steps (if any)
Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud) or similar.
1. Using the docs https://www.tiny.cloud/blog/get-started-with-tinymce-self-hosted/
2. Create folder for html testing.
3. Extracted tinymce so that path is js/tinymce/tinymce.min.js
4. Added index.html with:

`<script src="js/tinymce/tinymce.min.js"></script>
  <script type="text/javascript">
  tinymce.init({
    selector: '#mytextarea',
    license_key: 'gpl',
    toolbar_sticky: true,
    toolbar: 'bold italic blockquote | styles | bullist numlist | link unlink fullscreen',
    plugins: 'fullscreen autoresize',
    fullscreen_native: true
  });
  </script>
`

On the html page in the editor, after the "Hello World" create a number of empty lines until autoresize reaches the bottom and then some more so the "Hello World" disappears under the sticky toolbar. 

Then using the keyboard navigate up the editor and notice that "Hello World does not come back into view and the cursor itself disappears from view behind the sticky toolbar.

Navigation with the scroll wheel does work and reveals "Hello World".

### ✔️ Expected result
The expected result is that keyboard navigation up the editor will reveal all lines including the very top line "Hello World".

### ❌ Actual result
Described above.

## 📃 Other details
- Browser: Firefox and Edge
- OS: Windows 11
- First affected version: Does not work in 8.9.0 and may not have worked in previous versions.
- Worked in version: Unknown.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by izmeez** [comment]
When `toolbar_sticky` is used with `toolbar_sticky_offset` the number of lines that cannot be reached with keyboard navigation is the depth of the offset plus the depth of the toolbar.

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/aa20a3e4-7f83-4202-be3a-1e1836e3fe3a" />

And the scroll wheel cannot reach the top line.

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/fca9c94d-fd82-48fe-8ce0-5f94feee209a" />
